model/distro/distro.go

```diff
@@ -30,9 +30,10 @@ type Distro struct {
 	SSHKey      string   `bson:"ssh_key,omitempty" json:"ssh_key,omitempty" mapstructure:"ssh_key,omitempty"`
 	SSHOptions  []string `bson:"ssh_options,omitempty" json:"ssh_options,omitempty" mapstructure:"ssh_options,omitempty"`
 
-	SpawnAllowed bool        `bson:"spawn_allowed" json:"spawn_allowed,omitempty" mapstructure:"spawn_allowed,omitempty"`
-	Expansions   []Expansion `bson:"expansions,omitempty" json:"expansions,omitempty" mapstructure:"expansions,omitempty"`
-	Disabled     bool        `bson:"disabled,omitempty" json:"disabled,omitempty" mapstructure:"disabled,omitempty"`
+	SpawnAllowed   bool        `bson:"spawn_allowed" json:"spawn_allowed,omitempty" mapstructure:"spawn_allowed,omitempty"`
+	NoTasksAllowed bool        `bson:"no_tasks_allowed" json:"no_tasks_allowed,omitempty" mapstructure:"no_tasks_allowed,omitempty"`
```

Comment:
I don't think we need a new field - you can do the same thing by checking the HasContainers field in the host document

Reply:
Do I have access to the host document fields when I'm working in `distros.html`? I'm trying to think of how to set `HasContainers` via a checkbox.

Reply:
No, but my point is that this field isn't needed so you don't need to set it. The only change needed is the NeedsNewAgent query I think

Reply:
Removed new field and am now using `HasContainers` in the query